tests/test-override.sh

```diff
@@ -79,7 +79,11 @@ ${FLATPAK} override --user --nofilesystem=xdg-documents org.test.Hello
 ${FLATPAK} override --user --show org.test.Hello > override
 
 assert_file_has_content override "^\[Context\]$"
-assert_file_has_content override "^filesystems=/media;home;!xdg-documents;xdg-desktop/foo:create;xdg-config:ro;$"
+#assert_file_has_content override "^filesystems=.*/media;.*$"
```

Comment:
I don't get it. Why have a bunch of asserts commented out?

Reply:
that would be an oversight :(

they shouldn't be